FrameworkSystem/private/standardLogging/gLogging.py

```diff
@@ -0,0 +1,370 @@
+
+
+class gLogging(object):
+  """
+  gLogging is a wrapper of the logger object from the standard logging library which integrate
+  some DIRAC concepts. It is the equivalent to the old gLogger object.
+
+  It is used like an interface to use the logger object of the logging library. 
+  Its purpose is to replace transparently the old gLogger object in the existing code in order to 
+  minimize the changes. 
+
+  In this way, each gLogging embed a logger of logging. It is possible to create sublogger,
+  set and get the level of the embedded logger and create log messages with it.
+
+  gLogging could delegate the initialization and the configuration to a factory of the root logger be it can not
+  because it has to wrap the old gLogger.  
+  """
+
+  # componentName is a class variable because the component name is the same for every gLogging objects
+  # its default value is "Framework" but it can be configured and composed by the system name and the component name
+  _componentName = "Framework"
+
+  def __init__(self, father=None, fathername='', name=''):
+    """
+    Initialization of the gLogging object.
+    :params father: gLogging, father of this new gLogging.
+    :params fathername: string representing the name of the father logger in the chain.
+    :params name: string representing the name of the logger in the chain. 
+    By default, 'fathername' and 'name' are empty, because getChild accepts only string and the first empty
+    string corresponds to the root logger. 
+
+    Example: 
+    logging.getLogger('') == logging.getLogger('root') == root logger
+    logging.getLogger('root').getChild('log') == root.log == log child of root
+    """
+
+    # gLogging chain
+    self._children = {}
+    self._parent = father
+
+    # all the different backends
+    self._backendsDict = {'stdout': StdoutBackend(),
+                          'stderr': StderrBackend(),
+                          'file': FileBackend(),
+                          'server': RemoteBackend()}
+
+    # initialize display options and level with the ones of the gLogging parent
+    if self._parent is not None:
+      self._options = self._parent.getDisplayOptions()
+      self._level = LogLevels.getLevelValue(father.getLevel())
+    else:
+      self._options = {'showHeaders': True, 'showThreads': False, 'Color': False, 'Path': False}
+      # the native level is not used because it has to be to debug to send all messages to the log central
+      self._level = None
+
+    # name of the gLogging
+    self.name = name
+
+    # dictionary of the option state, modified by the user or not
+    # this is to give to the options the same behaviour that the logging level:
+    # - propagation from the parent to the children when their levels are not set by the developer himself
+    # - stop the propagation when a developer set a level to a child
+    self._optionsModified = {'showHeaders': False, 'showThreads': False}
+    self._levelModified = False
```

Comment:
This will not work no ? 
If for example I set the level of gLogging, then create a subLogger, and then reset the level of gLogging, the level will not be propagated to the subLogger